src/game/mario.c

```diff
@@ -512,6 +514,9 @@ u32 mario_get_terrain_sound_addend(struct MarioState *m) {
  * Determines if Mario is facing "downhill."
  */
 s32 mario_facing_downhill(struct MarioState *m, s32 turnYaw) {
+    if (m->floor && m->floor->type == SURFACE_SUPER_SLIPPERY)
+        return 0;
```

Comment:
i dont get what this is needed for, seems weird

Reply:
`return FALSE;` would be better for consistency here